### PR TITLE
Remove ctx from source structs

### PR DESCRIPTION
**Pull request by ahrav**

### CODE REVIEW

**Review by trufflesteeeve — APPROVED** [review]

File: proto/sources.proto [hunk 1]
```diff
@@ -157,6 +157,7 @@ message GitHub {
   string head = 9;
   string base = 10;
   repeated string ignoreRepos = 11;
+  repeated string includeRepos = 12;
```
Comment (mcastorina) [inline proto/sources.proto:160]: I think this shouldn't be part of this PR..
